.gitpod.yml

```
@@ -0,0 +1,9 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+- command: julia --version
+vscode:
+  extensions:
+    - julialang.language-julia
+    - colinfang.markdown-julia
+    - singularitti.vscode-julia-formatter
```

These extensions are not getting installed you have to give the following ids in order for them to be installed.

```suggestion
  extensions:
    - singularitti.vscode-julia-formatter@0.1.0:RmhU34gIXDaV/d9sKVGQWg==
    - colinfang.markdown-julia@0.0.1:7o0taaLFuuX28a/ksp1m7g==
    - julialang.language-julia@0.15.29:6iA33XMcDFiLuabO41qB5A==
```